meetings/2023-05-17.md

```diff
@@ -48,6 +73,23 @@ URL: https://github.com/solid/specification/pull/509
 * SC: Resolves https://github.com/solid/specification/issues/508
 * SC: Recommends "Use consistent spelling throughout the document." -- essentially it is Editors' responsibility. Rationale in https://github.com/solid/specification/issues/508#issuecomment-1461883028
 * SC: Any objections to merge?
+* TT: General guidance is good. The challenge is knowing what any doc is using. If you're reading something named in British spelling, you need to search documents on that thing and if they are using US spelling, you cannot find them. 
+* SC: For things that are terms that are defined,t hose will be whatever the initial definition is. 
+* TT: let's imagine there's a function "behaviour". You want to find all instances for that function but you have "behaviour" and "behavior". That's why the document is easy for the editor but maybe not the project. 
+* SC: So that is more about findability/searchability. If this issue would have been that significant it would have been a requirements. 
+* TT: The balance is to consider all the things and come up with the best suggestion. Manual of Style are "should". 
```

Comment:
```suggestion
* SC: For terms that are defined, those will be whatever the initial definition is. 
* TT: Let's imagine there's a function "behaviour". You want to find all instances of that function, but you have a mix of "behaviour" and "behavior". That's one way per-document consistency is easy for the editor but maybe not the project. 
* SC: So that is more about findability/searchability. If this issue would have been that significant, it would have been a requirements. 
* TT: The balance is to consider all the things and come up with the best suggestion. Manuals of Style are basically RFC "SHOULD". 
```